.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+	"arrowParens": "always",
+	"printWidth": 100,
+	"trailingComma": "all",
+	"singleQuote": true,
+	"useTabs": true
+}
```

Comment:
Do you want to make a prettier chore branch so core complete changes can be reviewed on their own?

Reply:
Yeah will do a full sweep after. I'm handling the merge conflcits with your changes here so won't impact you til this is merged.